src/regions/ca/info/faq/39_gloves.md.fr

```diff
@@ -0,0 +1,11 @@
+## Est ce qu’il faut que je mette des gants en latex ou vinyle lorsque je vais au supermarché?
+
+De la même manière que le port du masque, le port de gants offre également un faux sentiment de sécurité, ainsi qu’un risque de contamination lors d’une manipulation inapproprié.
+
+Elle est réservée à un usage par des professionnels de la santé ou de l’enfance lorsqu’il existe un risque immédiat avec des liquides biologiques.
+
+Le port des gants ne remplace en aucun cas l’hygiène des mains, et en cas d’utilisation, les gants doivent être utilisés le moins longtemps possible. L’intégrité des gant est sensible à l’utilisation de produits, au temps, à l’humidité de vos mains… Si vous portez des gants car vous vous occupez de quelqu’un d’infecté, il faut également vous lavez les mains avant et après les avoir utilisés.
+
+Voici comment prendre soin de quelqu’un d’infecté au COVID : https://www.canada.ca/fr/sante-publique/services/publications/maladies-et-affections/comment-prendre-soin-personne-atteinte-covid-19-maison-conseils-soignants.html
```

Comment:
Can we change this so it is not shown as a link in the page?

Reply:
Fixed!